### PR TITLE
Manual partitioning for Ubuntu

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

The current automatic partitioning does not work well with the automatic resizing of the root disk done by IM.

This is just a test with manual partitioning. However, there is a new problem: partitions do not resize automatically when booting the VM.

I am opening the PR to discuss further solutions.

---

**Related issue :** https://github.com/EGI-Federation/fedcloud-vmi-templates/issues/62

### CODE REVIEW

File: ubuntu/provisioners/roles/cloud-init/tasks/main.yaml [hunk 1]
```diff
@@ -6,18 +6,3 @@
...
 - name: distro specific cloud-init mods
   include_tasks: cloud-init-{{ansible_distribution_release }}.yaml
-
-- name: Create fedcloud config
-  copy:
-    content: |
-      # EGI FedCloud configuration
-      
-      # Make sure to disable ssh password authentication
-      ssh_pwauth: 0
```
Comment (enolfc) [inline ubuntu/provisioners/roles/cloud-init/tasks/main.yaml:16]: I would keep this in some way or another

File: ubuntu/httpdir/user-data [hunk 2]
```diff
@@ -14,8 +14,46 @@ autoinstall:
   ssh:
     install-server: true
   storage:
-    layout:
-      name: direct
```
Comment (enolfc) [inline ubuntu/httpdir/user-data:18]: Just remembered that I looked at this back in the day: https://github.com/NeCTAR-RC/nectar-images/blob/master/http/ubuntu-22.04/user-data#L18

Maybe we should read carefully how they manage on that repo

**Comment by enolfc** [comment]
> This is just a test with manual partitioning. However, there is a new problem: partitions do not resize automatically when booting the VM.

Maybe worth checking docs from cloud-init: https://cloudinit.readthedocs.io/en/latest/reference/modules.html#growpart and even the implementation

**Comment by sebastian-luna-valero** [comment]
This PR can be closed.

See resolution in:
* https://ggus.eu/?mode=ticket_info&ticket_id=165184#update#13
* https://github.com/grycap/im/issues/1523

Many thanks @astalosj for his time devoted to debug this!

**Pull request closed without merging**